aggregators/is-performant/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Aggregate = require('../aggregate');
```

Comment:
The way I'm looking at this is that we have these atomic tests, which we will probably want to batch up into various groups. You may want FMP to show up in performance, or some other group (aggregation) with different weighting. So, for example, let's say you have the short_name. In the context of the add to homescreen prompt aggregation it has zero weight, but for a manifest aggregation it will have a weight of one.